test/automl/test_forecast.py

```diff
@@ -572,10 +576,140 @@ def smape(y_pred, y_test):
     print(automl.min_resource)
 
 
+def test_cv_step():
+    n = 300
+    time_col = "date"
+    df = pd.DataFrame(
+        {
+            time_col: pd.date_range(start="1/1/2001", periods=n, freq="D"),
+            "y": np.sin(np.linspace(start=0, stop=200, num=n)),
+        }
+    )
+
+    def split_by_date(df: pd.DataFrame, dt: datetime.date):
+        dt = datetime.datetime(dt.year, dt.month, dt.day)
+        return df[df[time_col] <= dt], df[df[time_col] > dt]
+
+    horizon = 60
+    data_end = df.date.max()
+    train_end = data_end - datetime.timedelta(days=horizon)
+
+    train_df, val_df = split_by_date(df, train_end)
+    from flaml import AutoML
+
+    tgts = ["y"]
+    # tgt = "SERIES_SANCTIONS"
+
+    preds = {}
+    for tgt in tgts:
+        features = []  # [c for c in train_df.columns if "SERIES" not in c and c != time_col]
+
+        automl = AutoML(time_budget=5, metric="mae", task="ts_forecast", eval_method="cv")
+
+        automl.fit(
+            dataframe=train_df[[time_col] + features + [tgt]],
+            label=tgt,
+            period=horizon,
+            time_col=time_col,
+            verbose=4,
+            n_splits=5,
+            cv_step_size=5,
+        )
+
+        pred = automl.predict(val_df)
+
+        if isinstance(pred, pd.DataFrame):
+            pred = pred[tgt]
+        assert not np.isnan(pred.sum())
+
+        import matplotlib.pyplot as plt
+
+        preds[tgt] = pred
+        # plt.figure(figsize=(16, 8), dpi=80)
+        # plt.plot(df[time_col], df[tgt])
+        # plt.plot(val_df[time_col], pred)
+        # plt.legend(["actual", "predicted"])
+        # plt.show()
+
+    print("yahoo!")
+
+
+#
+#
+# def test_finance_df():
```

Comment:
Is the inclusion of the commented code intended?

Reply:
No, will remove, thanks.